doc/source/ray-air/examples/torch_detection.ipynb

```diff
@@ -823,19 +843,13 @@
                 "    },\n",
                 "    scaling_config=ScalingConfig(num_workers=4, use_gpu=True),\n",
                 "    datasets={\"train\": train_dataset},\n",
-                "    dataset_config={\n",
-                "        # Don't augment test images. Only apply `per_epoch_preprocessor` to the train\n",
-                "        # set.\n",
-                "        \"train\": DatasetConfig(\n",
-                "            per_epoch_preprocessor=per_epoch_preprocessor\n",
-                "        ),\n",
-                "    },\n",
```

Comment:
@ericl the original comment says “Don't augment test images.“, but there is only a train dataset passed in. Is this an error before?

Reply:
Yeah that seems out of date, we should probably remove the comment.